backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -277,6 +281,7 @@ and Dval =
   | DOption of Option<Dval>
   | DResult of Result<Dval, Dval>
   | DBytes of byte array
+  | DUserEnum of typeName : UserTypeName * caseName : string * fields : List<Dval>
```

Comment:
I'd call this DConstructor I think?

Reply:
:+1: please see comment above about "Constructor" feeling odd, though